mewarpx/mewarpx/mcc_wrapper.py

```diff
@@ -0,0 +1,90 @@
+import glob, os
+from pywarpx import picmi
+from mewarpx import util as mwxutil
+
+# For use later to sync with diode test template and access sim object in mwxrun
+from mewarpx import mwxrun
+
+
+class MCC():
+    """Wrapper used to initialize MCC parameters"""
+
+    def __init__(self, electron_species, ion_species, P_INERT,
+                 T_INERT, scraper=None, **kwargs):
+        """Initialize MCC"""
```

Comment:
You should add documentation here regarding the input arguments. What units are P_INERT & T_INERT in, etc.

Reply:
Since the capacitive discharge uses N_INERT directly, I think nice functionality would be accepting either P_INERT or N_INERT
```suggestion
    def __init__(self, electron_species, ion_species, T_INERT,
                 P_INERT=None, N_INERT=None, scraper=None, **kwargs):
        """Initialize MCC"""
        self.N_INERT = N_INERT
        if P_INERT is not None:
            if N_INERT is not None:
                raise ValueError("Must specify N_INERT or P_INERT, not both")
            self.P_INERT = P_INERT
            self.N_INERT = mwxutil.ideal_gas_density(self.P_INERT, self.T_INERT)
        elif N_INERT is None:
            raise ValueError("Must specify one of N_INERT or P_INERT")
```